native/dotnet-sdk-8.0/Makefile

```diff
@@ -0,0 +1,22 @@
+PKG_NAME = dotnet-sdk-8.0
+# Version 8.0.4, SDK 8.0.204
+# https://dotnet.microsoft.com/download/dotnet/8.0
+PKG_VERS = 8.0.204
```

Comment:
Latest dotnet sdk version is 8.0.301 - I've tested jellyfin 10.9.3 for compatibility and it works like a charm, shall we go to .301 right away?